common.blocks/issue/issue.js

```diff
@@ -36,13 +38,21 @@ provide(bemDom.declBlock(this.name, {
         var footer = this._elem('footer');
 
         event.preventDefault();
-        button.setMod('disabled');
 
-        request(button.params.number + '/comments')
-            .then(function(response) { return response.text(); })
-            .then(function(data) {
-                bemDom.append(footer.domElem, data);
-            });
+        if (!this.isCommentsOpened) {
+            this.isCommentsOpened = true;
+
+            request(button.params.number + '/comments')
+                .then(function(response) { return response.text(); })
+                .then(function(data) {
+                    bemDom.append(footer.domElem, data);
+                });
+        } else {
+            var comments = this.findChildBlock(Comments);
```

Comment:
you don't need this variable - just use
bemDom.destruct(this.findChildBlock(Comments).domElem);